src/quick-start.md

```diff
@@ -128,12 +165,12 @@ EOF
 This will produce the following output:
 `clusteradmissionpolicy.policies.kubewarden.io/privileged-pods created`
 
-Defining the `ClusterAdmissionPolicy` will lead to a rollout of the Kubewarden Policy
+Defining the `ClusterAdmissionPolicy` will lead to a rollout of the Kubewarden default Policy
```

Comment:
There is no policyServer in this ClusterAdmissionPolicy, so it will be scheduled in the default one

Reply:
True, but in the example above the policy is targeting a specific one.

Reply:
yes, you are right. But this is referring to the policy applied with kubectl that doesn't have policyServer which is in a different section than the other example _Enforce your first policy_ https://github.com/raulcabello/docs/blob/new-architecture/src/quick-start.md#enforce-your-first-policy
I would keep the other example with a policyServer to show how you can create a policy and bound it to a PolicyServer. But it would not add a policyServer to the _Enforce your first policy_  to make the quick start as simple as possible

Reply:
fine with me